solutions/bronze/usaco-739.mdx

```diff
@@ -377,5 +374,76 @@ public class BovineGenomics {
 	//CodeSnip{Kattio}
 }
 ```
+
 </JavaSection>
+<PySection>
+
+<Warning>
+
+Due to Python's slow speed, the below code TLEs on a couple test cases.
+
+</Warning>
+
+```py
+with open('cownomics.in') as read:
+	cow_num, genome_len = [int(i) for i in read.readline().split()]
+	
+	spotted = []
+	for _ in range(cow_num):
+		genome_str = read.readline()
+		genome = []
+		for g in range(genome_len):
+			if genome_str[g] == 'A':
+				genome.append(0)
+			elif genome_str[g] == 'T':
+				genome.append(1)
+			elif genome_str[g] == 'C':
+				genome.append(2)
+			elif genome_str[g] == 'G':
+				genome.append(3)
+		spotted.append(genome)
+
+	plain = []
+	for _ in range(cow_num):
+		genome_str = read.readline()
+		genome = []
+		for g in range(genome_len):
+			if genome_str[g] == 'A':
+				genome.append(0)
+			elif genome_str[g] == 'T':
+				genome.append(1)
+			elif genome_str[g] == 'C':
+				genome.append(2)
+			elif genome_str[g] == 'G':
+				genome.append(3)
```

Comment:
perhaps a lookup dictionary would be cleaner?

Reply:
this is how i did it in the java & c++ sol
not opposed to using a dict tho, just need someone else to weigh in